packages/figma-utils/src/variables/generate.ts

```diff
@@ -120,7 +139,8 @@ export const generateTimestampComment = (modeName?: string): string => {
  * @example "{your-variable-name}"
  * @returns `isAlias` whether the variable is an alias and `aliasName` the raw variable name without curly braces.
  */
-export const isAliasVariable = (variableValue: string) => {
+export const isAliasVariable = (variableValue?: string) => {
+  if (!variableValue) return { isAlias: false, aliasName: "" };
```

Comment:
Why can the variable value be undefined? Shouldn't they have been filtered out already / not exist at all?